src/Sylius/Bundle/CoreBundle/Resources/config/validation.xml

```diff
@@ -124,6 +124,17 @@
             </constraint>
         </property>
     </class>
+
```

Comment:
I saw that there are 2 same classname for Sylius\Component\Core\Model\Order in the file

Already exist at line 61

Is it the problem for failed test ?